pyproject.toml

```diff
@@ -89,19 +89,20 @@ dev = [
 ]
 docs = [
   "ipykernel",
-  "ipython",
+  "ipython >=8.5.0",
   "ipython_genutils",
   "jupyter_client",
-  "matplotlib >=3.5.2",
-  "nbconvert",
-  "nbsphinx",
-  "sphinx >=6.0.0",
+  "matplotlib >=3.6.0",
+  "nbconvert >=7.14.0",
+  "nbsphinx >=0.9.5",
+  "psy-maps >=1.5.0",
+  "sphinx >=7.0.0",
   "sphinx-rtd-theme >=1.0"
 ]
 extras = [
   "xesmf >=0.8.2",
   # See: https://github.com/pydata/xarray/issues/7794
-  "xarray >=0.21.0,<2023.03.0"
+  "xarray >=2022.6.0,<2023.3.0"
```

Comment:
xarray is in  `dependencies` _and_ in `optional-dependencies.extra` ?

Reply:
Yes, this is because the `xesmf` functions needs an _older_ `xarray` to run properly, otherwise cf-related problems occur. There's an issue to deal with this (@huard knows more), but it seems stalled on the `xarray` side.